### ISSUE TITLE
Two errors that I came across

### ISSUE THREAD

**Issue by imr555** [colinlaney/animal-tracking#6]
After pulling the latest version and running the code on the sample I came across this error

```
Traceback (most recent call last):
File "track.py", line 252, in <module>
floorCrop(filename)
File "track.py", line 91, in floorCrop
contours, hierarchy = cv2.findContours(mask, cv2.RETR_LIST, cv2.CHAIN_APPROX_SIMPLE)
ValueError: too many values to unpack (expected 2)
```

Adding `_,` and a space before line 91, like this 
`_, contours, hierarchy = cv2.findContours(mask, cv2.RETR_LIST, `cv2.CHAIN_APPROX_SIMPLE)`

I achieve another error sadly.This error comes after drawing a box around the rat,

`Segmentation fault (core dumped)`

**Comment by colinlaney** [comment]
@imr555, please pull new version.

* The 1st error is caused by version < 4.0.0.21 of your OpenCV, run `pip install -r requirements.txt`.
* The 2nd one is caused by conflict between `cv2.waitKey()` and `cv2.destroyWindow()`. It was okay on Windows but not on Mac OS X or Linux.

**Comment by imr555** [comment]
The previous issues are resolved. But theres a new error that has come up. After running the code twice each time at 120 sec, the program auto exits and this error shows up

```
Traceback (most recent call last):
  File "track.py", line 255, in <module>
    trace(filename)
  File "track.py", line 180, in trace
    x = int(M['m10']/M['m00'])
ZeroDivisionError: float division by zero
```

**Comment by colinlaney** [comment]
Yes, it's fixed too in [c560eb0](https://github.com/colinlaney/animal-tracking/commit/c560eb06dd9081f9bd00f8f474b909488d1c1b0e).
